keras_hub/src/models/retinanet/retinanet_label_encoder.py

```diff
@@ -24,17 +25,9 @@ class RetinaNetLabelEncoder(keras.layers.Layer):
     consistency during training, regardless of the input format.
 
     Args:
-        bounding_box_format: str. The format of bounding boxes of input dataset.
-            Refer TODO: Add link to Keras Core Docs.
-        min_level: int. Minimum level of the output feature pyramid.
-        max_level: int. Maximum level of the output feature pyramid.
-        num_scales: int. Number of intermediate scales added on each level.
-            For example, num_scales=2 adds one additional intermediate anchor
-            scale [2^0, 2^0.5] on each level.
-        aspect_ratios: List[float]. Aspect ratios of anchors added on
-            each level. Each number indicates the ratio of width to height.
-        anchor_size: float. Scale of size of the base anchor relative to the
-            feature stride 2^level.
+        anchor_generator:  A `keras_hub.layers.AnchorGenerator`.
+        bounding_box_format: str. TODO:
```

Comment:
you can still add a line of description here for the arg.

Reply:
Done